REFERENCE.md

```diff
@@ -20,10 +20,6 @@
 * [`extlib::resources_deep_merge`](#extlibresources_deep_merge): Deeply merge a "defaults" hash into a "resources" hash like the ones expected by `create_resources()`.
 * [`extlib::sort_by_version`](#extlibsort_by_version): A function that sorts an array of version numbers.
 
-**Facts**
```

Comment:
This section should have never been added here (which is must have been manually).
I'll move these docs into https://github.com/voxpupuli/puppet-extlib/pull/152